### DISCUSSION TITLE
Prohibit direct access to url /admin/files

### DISCUSSION THREAD

**Discussion by Pachat** [directus/directus#19831] [Feature Requests]
### Summary

Even if I uncheck Files library, a user can still add `/files` to the URL and see all files.

But I want the user to have access to files only through collections. Ex. : the avatar on the profile.
And for other collections with restricted items, only to the files that are related to the items he has access to.
That is, only in the detail item he has access to.

How maintain privacy?

### Basic Example

Not see the photos of all members but only those of his group.

### Motivation

Maintain private access upon the files

### Detailed Design

Don't know

### Must Have:
- A
- B

### Should Have:
- C

### Could Have:
- D

### Won't Have:
- E

### Drawbacks

None when what you want if privacy

### Alternatives

No

### Adoption Strategy

Easy

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
